src/adapter/src/coord/timestamp_selection.rs

```diff
@@ -451,6 +453,32 @@ impl Coordinator {
                 &compute_instance.to_string(),
             ])
             .inc();
+        if isolation_level == &IsolationLevel::StrictSerializable && real_time_recency_ts.is_none()
```

Comment:
Thoughts about if this should also limit by `can_respond_immediately=false` because if that's true then this number isn't interesting.

Reply:
yea good point